### PR TITLE
Rails Blog App: (1) Creating Data Model

### PR DESCRIPTION
**Pull request by adarachel**
### In this Milestone, I implemented the following Project Requirements:

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
       - Create and run the necessary migration files.
       - Table and column names should match the ERD diagram.
       - note: photo for users table should be a link to an image
- Foreign keys should be included.
      - Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
      - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20230819054513_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
```
Comment (mikethreels) [inline db/migrate/20230819054513_create_users.rb:6]: - [x] in the user table there should be a column for post_counter according to the requirements

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Rachel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job creating the tables

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230819054513_create_users.rb (hunk 1 above)
Comment (Whoistolu) [reply db/migrate/20230819054513_create_users.rb:6]: - Well resolved

**Review by Whoistolu — APPROVED** [review]
### Hi @adarachel 

Great job implementing the recommendations by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**